lib/src/util/react_wrappers.dart

```diff
@@ -280,6 +280,43 @@ react.Component getDartComponent(/* [1] */ instance) {
     return null;
   }
 
+  // Split out into a separate function since the DDC doesn't support
```

Comment:
This actually isn't needed now that I understand the issue more. See https://github.com/dart-lang/sdk/issues/30426